### DISCUSSION TITLE
Remove a page icon

### DISCUSSION THREAD

**Discussion by adcousin** [ramnes/notion-sdk-py#128] [Q&A] [answered]
Good day, and first of all thanks for the sdk.
Is it possible to remove a page icon with the current branch ?

I've tried `notion.pages.update('my_page_id', **{'icon': None})` with `my_page_id` having an icon. I get a 200 result but the icon remains.
If I remove the icon manually from UI and retrieve my_page_id, I get `'icon: None`.

Regards.

**Comment by FlorianWilhelm** [comment]
@adcousin, I got exactly the same problem and could not find a workaround. The question is if this is a bug in the Notion API or if `notion-sdk-py` is stripping out the `null` in the request.

**Comment by FlorianWilhelm** [comment]
@adcousin, PR #220 fixes your problem.

BTW: Check out also [Ultimate Notion](https://ultimate-notion.com/) :-)

**Comment by ramnes** [comment, marked as answer]
Should be fixed in release 2.2.0 🙏
